### PR TITLE
Create notes-2019-12-16.md

### PR DESCRIPTION
**Pull request by romulocintra**
Added meeting notes for 2019-12-16 Meeting

cc @sffc

### CODE REVIEW

File: meetings/notes-2019-12-16.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+TCQ: https://tcq.app/meeting/8zpZ
```
Comment (sffc) [inline meetings/notes-2019-12-16.md]: Remove the link to TCQ for now

File: meetings/notes-2019-12-16.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+#### December Attendees:
+Please fill in a 3-letter acronym if this is your first meeting:
```
Comment (sffc) [inline meetings/notes-2019-12-16.md]: Remove the instructions

File: meetings/notes-2019-12-16.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+- Staś Małolepszy (SMY)
+- Zibi Braniecki - Mozilla (ZB)
+- Mihai Nita - Google (MIH)
+- Rafael Xavier - PayPal (RX)
+- Staś Małolepszy - Mozilla (SMY)
+- Shane Carr - Google (SFC)
+- Elango Cheran - Google (ECH)
+- Long Ho - Dropbox (LHO)
+
+#### Last Meeting Attendees:
```
Comment (sffc) [inline meetings/notes-2019-12-16.md]: Remove this section

**Comment by romulocintra** [comment]
Will do same
Changes
Here

**Pull request merged**